templates/repo/actions/view.tmpl

```diff
@@ -0,0 +1,10 @@
+{{template "base/head" .}}
+
+<div class="page-content repository">
+	{{template "repo/header" .}}
+	<div id="repo-build-view" data-run-index="{{.RunIndex}}" data-job-index="{{.JobIndex}}">
+	</div>
```

Comment:
```suggestion
    <div class="ui container">
        <div id="repo-build-view" data-run-index="{{.RunIndex}}" data-job-index="{{.JobIndex}}">
        </div>
    </div>
```

to limit width.

Reply:
Actually, it's designed that way on purpose. You'll see it's too crowded with limited width:

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/9418365/206415167-b73ecb8d-0f9c-4a8e-857d-cf4890436684.png">
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/9418365/206415267-bf54f8fd-43d1-491c-b38a-777b4c892e8a.png">

Reply:
![image](https://user-images.githubusercontent.com/25342410/206448948-b21b1c80-045f-492e-8b60-7c4c4bad9ebe.png)
but it's really too width now, maybe at least add some pading ...

Reply:
Solved.